pkg/printers/sarif.go

```diff
@@ -221,6 +215,19 @@ func sarifEntriesFromJSONObject(jsonObject interface{}, pathExpressions map[stri
 	return sarifEntries, nil
 }
 
+func validateDataEntries(entries map[string][]string) (int, error) {
+	numberOfValues := len(entries[SarifRuleJSONPathExpressionKey])
+	for key, values := range entries {
+		// "-" is used as an empty replacement value in case values are missing in an array for GJSON. Hence, ignore
+		// all values which may not match the number of expected values iff the array contains the replacement value.
+		if len(values) != numberOfValues && !slices.Equal(values, []string{"-"}) {
```

Comment:
Probably due to the old golang version on the release branch I assume?
```suggestion
		if len(values) != numberOfValues && !sliceutils.Equal(values, []string{"-"}) {
```